### ISSUE TITLE
Horus copies @change, and other @ tags, to merged sentences

### ISSUE THREAD

**Issue by morbrorper** [Tatoeba/tatoeba2#3088]
If a sentence that has a `@change` tag is changed, and the change results in a merge with another sentence, the tag is copied to the remaining sentence. This leads to a confusing state that can only be corrected by the owner of the tag, if they happen to notice, or an admin.

Of course, this applies to all @ tags, such as `@needs_native_check`.

Suggested solution: Make Horus exclude tags prefixed with "@" in copying tags when merging sentences.

**Comment by morbrorper** [comment]
> Related: #2595 It's closed

Yes, related, but not identical, as this only concerns special tags, prefixed with "@".

**Comment by ckjpn** [comment]
[AndiPersti](https://github.com/AndiPersti) comment further refers to this comment on another issue that is related.

https://github.com/Tatoeba/tatoeba2/issues/1830#issuecomment-583591293
